batch7crud-roll1/src/main/java/com/lftechnology/batch7crud/database/DatabaseConnection.java

```diff
@@ -0,0 +1,36 @@
+package com.lftechnology.batch7crud.database;
+
+import com.lftechnology.batch7crud.exception.DataException;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+
+public class DatabaseConnection {
+
+	public static Connection getConnection()
+					throws DataException {
+
+		try {
+
+			Context initContext = new InitialContext();
+			Context envContext = (Context) initContext.lookup("java:/comp/env");
+			DataSource dataSource = (DataSource) envContext.lookup(
+							"jdbc/lf_testdatabase");
+			Connection connection = dataSource.getConnection();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "connection". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)